src/Compilers/VisualBasic/Portable/Utilities/FeatureUtils.vb

```diff
@@ -0,0 +1,53 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Option Compare Binary
+Option Strict On
+
+Imports Microsoft.CodeAnalysis.VisualBasic
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
+
+    Friend Class FeatureUtils
```

Comment:
I made them `class` and `friend` so that I didn't have to go through a public api review.